content/en/docs/contribute/style/style-guide.md

```diff
@@ -46,20 +46,24 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 When you refer specifically to interacting with an API object, use [UpperCamelCase](https://en.wikipedia.org/wiki/Camel_case), also known as Pascal Case. When you are generally discussing an API object, use [sentence-style capitalization](https://docs.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization).
 
+You may use the word "resource", "API", or "object" to clarify a Kubernetes resource type in a sentence. Note that Kubernetes resource names are capitalized. However, capitalization alone is not a reliable way to identify Kubernetes resource types in a sentence. 
+
+The sentence "However, pods may have multiple volume mounts" describes general concepts, and uses lower case. In contrast, the sentence "The Pod object includes a Volumes field" is talking about API objects, and uses UpperCamelCase. You may also use [Code Style](#code-style) to distinguish API objects. For example, "The `Pod` object includes a `volumes` field."
```

Comment:
```suggestion
The sentence "However, pods may have multiple volume mounts" describes general concepts, and uses lower case. In contrast, the sentence "The Pod object includes a volumes field" is talking about the Kubernetes API, and uses UpperCamelCase. To distinguish fields and resource names from natural language, use [Code Style](#code-style) whereever there is any risk of confusion. For example, "The `Pod` object includes a `.volumes` field."
```

Reply:
damn

Reply:
?

Reply:
- Since when we agreed to include a "." in the field name? Is this an improvement or an intent to drive this astray?
- Is "whereever" a newly invented English word?
- The proposed change is about "API objects", which is very clear. Why are we trying to mess this with "Kubernetes API"?

Reply:
I would leave out the information starting on line 51.

Reply:
I am going with @kbhawkey 's suggestion to remove line 51 and keep this PR focused. I will save a copy of the proposed changes for further consideration.